lib/util/cache/repository/impl/local.ts

```diff
@@ -1,27 +1,13 @@
-import { promisify } from 'util';
-import zlib from 'zlib';
-import hasha from 'hasha';
 import upath from 'upath';
 import { GlobalConfig } from '../../../../config/global';
 import { logger } from '../../../../logger';
 import { outputCacheFile, readCacheFile } from '../../../fs';
-import {
-  CACHE_REVISION,
-  isValidRev10,
-  isValidRev11,
-  isValidRev12,
-} from '../common';
 import type { RepoCacheRecord } from '../types';
-import { RepoCacheBase } from './base';
+import { RepositoryCacheBase } from './repository-cache-base';
 
-const compress = promisify(zlib.brotliCompress);
-const decompress = promisify(zlib.brotliDecompress);
-
-export class LocalRepoCache extends RepoCacheBase {
-  private oldHash: string | null = null;
-
-  constructor(private platform: string, private repository: string) {
-    super();
+export class LocalRepositoryCache extends RepositoryCacheBase {
```

Comment:
the word "repository" a bit annoying in this case
maybe 

- RepositoryCacheLocal
- RepositoryLocalCache

Reply:
agree, renamed all to use the original naming scheme files included.